cluster/network.go

```diff
@@ -140,6 +141,7 @@ func (c *Cluster) doCalicoDeploy(ctx context.Context) error {
 	calicoConfig := map[string]string{
 		KubeCfg:       clientConfig,
 		ClusterCIDR:   c.ClusterCIDR,
+		NodePortRange: c.Services.KubeAPI.ServiceNodePortRange,
```

Comment:
what is reason behind configuring the node port range?

Reply:
In the Calico doc: https://docs.projectcalico.org/v3.1/usage/enabling-ipvs
"Calico needs to be configured with the port range that is assigned to Kubernetes NodePorts. If services do use NodePorts outside Calico’s expected range, Calico will treat traffic to those ports as host traffic instead of pod traffic"